omnigan/blocks.py

```diff
@@ -136,6 +157,80 @@ def __str__(self):
         return strings.resblock(self)
 
 
+_BOTTLENECK_EXPANSION = 4
```

Comment:
what is this global variable for? is this something we'll never control for?

Reply:
Yes somehow in the source code of Advent they put it as a global variable but it's always 4 in the "final" deeplab architecture

Reply:
ok